help/tquery.md

```diff
@@ -0,0 +1,52 @@
+# Tquery
+
+### Typy danych i operatory filtrów
+
+| typ          | null(5) | = | in   | cmp | like | select | sort |
+|--------------|---------|---|------|-----|------|--------|------|
+| bool         | *       | * | .    | .   | .    | *      | +    |
+| date         | *       | * | *    | *   | .    | *      | +    |
+| datetime     | *       | . | .    | *   | .    | *      | +    |
+| int          | *       | * | *    | *   | *    | *      | +    |
+| string       | *       | * | *    | *   | *    | *      | +    |
+| uuid         | *       | * | *    | .   | .    | *      | .    |
+| text         | *       | . | .    | .   | *    | *(8)   | .    |
+| --------(9)  |         |   |      |     |      |        |      |
+| dict(1)      | *       | * | *    | .   | .    | *      | *    |
+| object(2)    | *       | . | .    | .   | .    | *      | .    |
+| dict_list(3) | *       | * | *(7) | .   | .    | *      | .    |
+| list(4)      | *       | . | .    | .   | .    | *      | .    |
+
+- `null` - operator poprawny tylko dla typów nullable
+- `=` - porównanie z wartością tego samego typu, int-int, string-string, itp., brak możliwości porównania z null ani z
+  pustym stringiem
+- `==`(?) - porównanie typu binary, bardziej kosztowne dla sql'a, ale z zachowaniem wielkości liter, polskich liter,
+  białych znaków
+- `in` - zawiera się w niepustej liście, reguły jak dla `=`
+- `cmp` - `<`, `<=`, `>`, `>=`, reguły typów jak dla `=`
+- `like` - wartość to niepusty string
+  - `%v`, `v%`, `%v%` - w wartości `%` i `_` są escape'owane
+  - możliwy `v`, gdzie wartość nie jest escape'owana
+
+
```

Comment:
Brakuje mi jeszcze informacji co się dzieje z polami null. Spodziewam się że wszystkie filtry poza null odfiltrowują nulle, tak? Czyli przykładowo filtr `inv:true,op:"=",val:"x"` nie zwróci nulli w tej kolumnie, tak? Dopisz to tutaj.

Wydaje się że tak byłoby logicznie jeśli wziąć pod uwagę jak baza traktuje takie wyrażenia. 
Natomiast jest to raczej głupie z punktu widzenia użytkownika, który nic nie wie o nullach, tylko sądzi że tam jest pusta wartość, a pusta wartość jest różna od "x". Przy czym ten problem UX powstaje raczej tylko dla filtrów z inv:true a w tej chwili takich się nie da wyklikać, a zresztą oczywiście frontend, jak mu nie pasuje, to może dohackować tam warunek "albo null".